### DISCUSSION TITLE
Displays errors and warnings in the problems window.

### DISCUSSION THREAD

**Discussion by UladShumeika** [microsoft/vscode-cmake-tools#3504] [Q&A] [answered]
Hi all. 
Previously, I had a project in VS 2022. I transferred its assembly to cmake and am working through VScode. However, I don't like that errors and warnings are simply displayed in the terminal and are not duplicated in the problems window. Please tell me, is it possible to somehow customize the output or edit the parsing of errors output by MSBuild?

![mtXvlj8](https://github.com/microsoft/vscode-cmake-tools/assets/77583317/73b4ac9d-bf22-48a1-b402-74a57cbd65bb)

**Comment by csorvagep** [comment, marked as answer]
Hi, It is possible, if you create a task for it. [VS Code tasks](https://code.visualstudio.com/docs/editor/tasks) are able to use error parsers and show them in the problems section. Although this is independent of the cmake-tools extension. If you set up a CMake project then you automatically get this.
